Content.Server/StationEvents/Metric/StationMetric.cs

```diff
@@ -0,0 +1,70 @@
+using Content.Server.GameTicking;
+using Content.Server.StationEvents.Metric.Components;
+using Content.Shared.chaos;
+
+namespace Content.Server.StationEvents.Metric;
+
+public abstract class StationMetric<T> : EntitySystem where T : Component
```

Comment:
**Right now (before a rename):**
A `StationMetric` is a system baseclass for Metrics which each read other components in the station and calculates a metric. 
A `StationMetricSystem` instantiates the StationMetrics and polls them for info.

This is like a `GameRule` which is also a `EntitySystem` - for instance `KudzuGrowthRule` is technically a system but since it ends with the Rule Suffix we already know that.

1. Can I call a given metric for instance `CombatMetric` ?
2. Their baseclass, you think it should be `StationMetricSystem`?
3. Not sure what to name their manager, which instantiates them but is itself a system. `StationMetricManager`?

Thanks for your help with naming.